roles/blobber/defaults/main.yml

```diff
@@ -0,0 +1,47 @@
+blobber_user: blobber
+
+blobber_cleanup: false # when set to "true" it will remove the container(s)
+
+################################################################################
+##
+## blobber container configuration
+##
+################################################################################
+blobber_container_name: blobber
+blobber_container_image: parithoshj/blobber:v0.0.1
+blobber_container_env: {}
+blobber_log_level: info
+
+# Network configs
+blobber_p2p_port: 20000
+blobber_announced_ip: "{{ ansible_host }}"
```

Comment:
Var not used anywhere. Probably can be removed?

Reply:
fixed